### DISCUSSION TITLE
Time is shifted in 1 second - Pressure

### DISCUSSION THREAD

**Discussion by fabo94** [DualSPHysics/DualSPHysics#237] [DUALSPHYSICS_V5.2]
Hello DualSPHysics community, 

I am modeling a tsunami-like wave propagating on a large flume and impacting a specimen that resembles an elevated bridge. 
When I correlate the numerical solutions with the data recorded during the experimental campaign, I have nice agreement for the wave characteristics (elevation and velocity), but when computing the pressure on the specimen, the time is always shifted in 1 second. 
Does anyone else experienced this issue before? Any suggestions?

**Comment by alexbexe** [comment]
Please check if you have frozen any rigid body for 1 second in the XML (FtPause)

Regards
